### DISCUSSION TITLE
Please integrate GraphRAG

### DISCUSSION THREAD

**Discussion by Tanguosen** [langgenius/dify#8087] [Suggestion]
As we all know，GraphRAG is in the spotlight，and if it can be integrated,Dify will be even more popular.

**Comment by AAEE86** [comment]
GraphRAG is still in the demo stage and is not mature.

**Comment by brightwang** [comment]
I created a video about how to build an AI agent by integrating graphrag and Dify.

https://youtu.be/cUtaEkIc65g?si=UG1QA3R-FvVEZwa7

**Reply by javilazar** [reply]
nice! any option to have to in English?
